tests/ffi/modules/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+exec_cmd "podman run -d --rm --replace -d --name ffi-asil \
+          quay.io/centos-sig-automotive/ffi-tools:latest  > /dev/null"
+
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+if [ ! -d "${QM_HOST_REGISTRY_DIR}" ]; then
+    exec_cmd "mkdir -p ${QM_HOST_REGISTRY_DIR}"
+    exec_cmd "podman push ${image_id} dir:${QM_HOST_REGISTRY_DIR}/ffi-tools:latest"
+fi
+
+podman exec -it qm /bin/bash -c \
+         "podman run  --replace --name ffi-qm  dir:${QM_REGISTRY_DIR}/ffi-tools:latest \
+         ./modprobe_module > /dev/null"
+
+if [ $? -eq 137 ]; then
```

Comment:
the same question I did in PR #448. Could you please check there ?

Reply:
+2 here, I think you need to catch the desired error

Reply:
Remove it. It is unnecessary.